crates/bevy_pbr/src/light_probe/generate.rs

```diff
@@ -0,0 +1,1084 @@
+//! Generated environment map filtering.
```

Comment:
For the systems in this file, try to step back and see what they'd look like without context. Remember that the names show up in stuff like tracy. So I think more descriptive names like prepare_generated_environment_map_foo is helpful.

Reply:
resolved by https://github.com/mate-h/bevy/pull/8